### ISSUE TITLE
Open in Panel

### ISSUE THREAD

**Issue by psychemedia** [jupyterhub/jupyter-server-proxy#468]
Currently, proxied apps appear to be opened into a new browser tab from a launcher button.

In contrast, [`upyterlab-pytutor`](https://github.com/jupyterlab-contrib/jupyterlab-pytutor) is opened from a JupyterLab notebook toolbar and opened into a panel, with code from one or more selected code cells passed into the iframe via a an encoded iframe URL parameter.

<img width="876" alt="image" src="https://github.com/jupyterhub/jupyter-server-proxy/assets/82988/27eb5643-1564-4fc4-9832-b7f424a26f8c">

Poking through the code, the extension creates a [panel](https://github.com/jupyterlab-contrib/jupyterlab-pytutor/blob/cbda37bd36d196bde28eb9bcd3678bb156aaaad2/src/index.ts#L150) that contains an [iframe](https://github.com/jupyterlab-contrib/jupyterlab-pytutor/blob/cbda37bd36d196bde28eb9bcd3678bb156aaaad2/src/index.ts#L181).

In some situations, it might be useful to be able to open a proxied service in to a panel, perhaps defined via a config setting:

```python
{
    'command': [...],
    'environment': ...,
    'timeout': ...,  

    'launcher_target': 'tab' | 'panel_iframe' 

    'launcher_entry': {
        'title': ...
        'icon_path': ...
     }
}
```

Clicking on the launcher with an iframe target would then open the created URL in a iframe in a panel inside JupyterLab.

**Comment by ryanlovett** [comment]
[new_browser_tab](https://jupyter-server-proxy.readthedocs.io/en/latest/server-process.html#new-browser-tab) implemented by #193 controls whether apps launch in a new browser tab or new jupyterlab tab. @cmd-ntrf created https://youtu.be/VFmIy9Wb83k to demonstrate how this works.

Is this option sufficient for you?

`new_browser_tab` is documented to be False by default, but it is [actually True](https://github.com/jupyterhub/jupyter-server-proxy/blob/main/jupyter_server_proxy/config.py#L152). (!)

**Comment by ryanlovett** [comment]
> `new_browser_tab` is documented to be False by default, but it is [actually True](https://github.com/jupyterhub/jupyter-server-proxy/blob/main/jupyter_server_proxy/config.py#L152). (!)

This was fixed by #470 .

**Comment by psychemedia** [comment]
Ah, thanks, missed that in the docs; thanks.
